src/bin/hexyl.rs

```diff
@@ -296,8 +294,10 @@ fn run() -> Result<()> {
 
     let show_color = match matches.get_one::<String>("color").map(String::as_ref) {
         Some("never") => false,
-        Some("auto") => std::io::stdout().is_terminal(),
-        _ => true,
+        Some("always") => true,
+        _ => supports_color::on_cached(supports_color::Stream::Stdout)
```

Comment:
minor: we only call it once, right? so we could use the non-caching version.
```suggestion
        _ => supports_color::on(supports_color::Stream::Stdout)
```